source/includes/gcp/_instance_groups.md

```diff
@@ -1 +1,199 @@
-### Instance groups
\ No newline at end of file
+### Instance groups
+An instance group is a collection of [VM instances](#gcp-instances) that you can manage as a single entity. Instance groups are used to apply [load balancing](#gcp-load-balancing) to groups of heterogeneous instances. In a load balancer setup, an instance group can be used as part of a [backend service](#gcp-backend-service).
```

Comment:
Mardown linter complains if there is no new line after a header

Reply:
let me fix this. Thanks.

Reply:
Fix linting issues.